src/lib/OpenEXRCore/decompress.c

```diff
@@ -154,13 +154,13 @@ undozip (
     void*                               out,
     size_t                              outsz)
 {
-    uLongf outSize = outsz;
+    uLongf outSize = (uLongf)outsz;
```

Comment:
since this a do-over, now might be a time to replace uLongF with a sized type :) Every time I see uLongf, I have to go grep for what it is!